src/WingetCreateCLI/Commands/NewCommand.cs

```diff
@@ -379,12 +379,9 @@ private async Task<bool> PromptPackageIdentifierAndCheckDuplicates(Manifests man
         {
             GitHub client = new GitHub(this.GitHubToken, this.WingetRepoOwner, this.WingetRepo);

```

Comment:
Shouldn't this call CheckForGitHubToken too? This seems to be the same circumstance as in UpdateCommand.cs

Reply:
No longer needed anymore, see LoadGitHubClient() method in base command class.